test/jdk/sun/security/pkcs11/PKCS11Test.java

```diff
@@ -715,7 +715,7 @@ static byte[] generateData(int length) {
         return data;
     }
 
-    private static Path fetchNssLib(String osId, Path libraryName) {
+    private static Path fetchNssLib(String osId, Path libraryName) throws IOException {
```

Comment:
Is the `IOException` caught later and wrapped as a `SkippedException`?

Reply:
I changed `ArtifactResolver.fetchOne()` to throw a SkippedException but this method still throws an IOException if the nss directory/artifact doesn't contain the necessary files. That case is handled by different tests and I didn't want to change it.